Njord/src/main/java/frc/robot/subsystems/ElevatorSubsystem.java

```diff
@@ -23,15 +38,56 @@ public class ElevatorSubsystem extends SubsystemBase {
   public enum ElevatorState {
     S_GoingUp, S_GoingDown, S_Stopped
     }
+
+  private final MutVoltage mVoltage = Volts.mutable(0);
+  private final MutDistance mDistance = Meters.mutable(0);
+  private final MutLinearVelocity mVelocity = MetersPerSecond.mutable(0);
+
+private Voltage setVolts = (Voltage volts) -> setVoltage(volts);
+private double setLog = (SysIdRoutineLog) -> setLog(SysIdRoutineLog);
+
+
+
+private final SysIdRoutine mSysIdRoutine = 
+   new SysIdRoutine(
+     new SysIdRoutine.Config(Volts.per(Seconds).of(1.2), Volts.of(12), Seconds.of(10)),
+     new SysIdRoutine.Mechanism(
+      voltage -> {
+        elevatorMotor1.setVoltage(voltage);
+      },
+        log -> {
+        log.motor("drive")
+        .voltage(
+         mVoltage.mut_replace(
+           elevatorMotor1.get() * RobotController.getBatteryVoltage(), Volts)
```

Comment:
This doesn't seem right. I don't think we need to multiple by battery voltage. Not sure what that is here.

```suggestion
           elevatorMotor1.get(), Volts)
```

Reply:
I'm incorrect. You're right. 

Ignore this suggestion.